en-US/emails/2018/jan_mobile_app_updates.lang

```diff
@@ -0,0 +1,77 @@
+## NOTE: Campaign for Mobile Apps Update Email
+## NOTE: Bug 1433498
+
+
+# Subject
+;Browse anywhere: Get Firefox on your new mobile devices
+Browse anywhere: Get Firefox on your new mobile devices
+
+
+# Pre-header
+;Plus: Protect your privacy, save your faves and get Firefox for Fire TV.
+Plus: Protect your privacy, save your faves and get Firefox for Fire TV.
+
+
+# Headline
+;Put Firefox on all your new toys
+Put Firefox on all your new toys
+
+
+;Getting a shiny new device can bring out the kid in all of us. But in your excitement, don't forget the same speed and privacy you love and expect from Firefox isn’t just available on your laptop or desktop.
+Getting a shiny new device can bring out the kid in all of us. But in your excitement, don't forget the same speed and privacy you love and expect from Firefox isn’t just available on your laptop or desktop.
+
+
+;So why not get Firefox on your phone or tablet, too? You can sign in to your Firefox Account and your open tabs, bookmarks and passwords will be waiting for you.
+So why not get Firefox on your phone or tablet, too? You can sign in to your Firefox Account and your open tabs, bookmarks and passwords will be waiting for you.
+
+
+;Want more Firefox in more places? You’re in luck:
+Want more Firefox in more places? You’re in luck:
+
+
+# This refers to Firefox Focus.
+;Automatically blocks ads and trackers to keep you safe and browsing fast.
+Automatically blocks ads and trackers to keep you safe and browsing fast.
+
+
+# It links users to the app.
+;Get Firefox Focus »
+Get Firefox Focus »
+
+
+# This refers to Pocket.
+;Save articles, videos and more, then read or watch them anywhere without ads, even offline.
+Save articles, videos and more, then read or watch them anywhere without ads, even offline.
+
+
+# It links users to the app.
+;Get Pocket »
+Get Pocket »
+
+
+# Bold text
+;Firefox for Fire TV:
+Firefox for Fire TV:
+
+
+;Firefox keeps you in cat videos — or whatever you’re into — even when there’s no app available.
+Firefox keeps you in cat videos — or whatever you’re into — even when there’s no app available.
+
+
+# It links users to the app.
+;Get Firefox for Fire TV »
+Get Firefox for Fire TV »
+
+
+# Signoff
+;Have fun and stay safe,
+Have fun and stay safe,
+
+
+;The Firefox Team
+The Firefox Team
+
+
+# The link points to https://support.mozilla.org/.
+;P.S. Questions? Problems? Concern? <a>Check out our support site.</a>
+P.S. Questions? Problems? Concern? Check out our support site.</a>
```

Comment:
Source string and translation are supposed to be identical ;-)

Fixed that, feel free to add other languages.

Reply:
gosh, first i missed the string, then this. :(.